packages/roosterjs-editor-types/lib/interface/ImageEditOptions.ts

```diff
@@ -63,4 +64,10 @@ export default interface ImageEditOptions {
      * Whether image crop is disabled. @default false
      */
     disableCrop?: boolean;
+
+    /**
+     * Which operations will be executed when image is selected
+     * @default ImageEditOperation.ResizeAndRotate
+     */
+    onSelectState?: ImageEditOperation;
```

Comment:
Use both ImageEditOperation and CompatibleImageEditOperation here:
```ts
onSelectState?: ImageEditOperation | CompatibleImageEditOperation;
```

Reply:
Added